src/cli_plugin/install/cleanup.js

```diff
@@ -0,0 +1,32 @@
+import rimraf from 'rimraf';
+import fs from 'fs';
+
+export function cleanPrevious(settings, logger) {
+  return new Promise(function (resolve, reject) {
+    try {
+      fs.statSync(settings.workingPath);
+
+      logger.log('Found previous install attempt. Deleting...');
+      try {
+        rimraf.sync(settings.workingPath);
+      } catch (e) {
+        reject(e);
+      }
+      resolve();
+    } catch (e) {
+      if (e.code !== 'ENOENT') reject(e);
+
+      resolve();
+    }
+  });
+};
+
+export function cleanError(settings) {
+  // delete the working directory.
+  // At this point we're bailing, so swallow any errors on delete.
+  try {
+    rimraf.sync(settings.workingPath);
+    rimraf.sync(settings.plugins[0].path);
+  }
+  catch (e) {} // eslint-disable-line no-empty
```

Comment:
I'm not entirely sure when this is used, but would be it make sense to at least log errors here to stderr, or do we expect this to error frequently enough in practice for that to be more annoying than helpful?